catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1522,6 +1525,85 @@ public UpdateResponse update(UpdateRequest updateRequest)
         return updateResponse;
     }
 
```

Comment:
These two functions are extremely similar yet have no common interface ground. Any ideas on how to abstract this?